eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/ReplicationEndpoint.scala

```diff
@@ -363,6 +359,96 @@ class ReplicationEndpoint(
 }
 
 /**
+ * An [[EndpointFilter]] computes a [[ReplicationFilter]] that shall be applied to a
+ * replication read request that replicates from a source log (defined by ``sourceLogName``)
+ * to a target log (defined by ``targetLogId``).
+ */
+trait EndpointFilter {
+  def filterFor(targetLogId: String, sourceLogName: String): ReplicationFilter
+}
+
+object EndpointFilter {
```

Comment:
To be more consistent with existing variable names (and documentation) `EndpointFilter` should be renamed to `EndpointFilters`. However, I would prefer the name `ReplicationFilterCombination` or `ReplicationFilterCombiner` or somthing similar. WDYT?